subgraph/tests/matchstick/access-control/CollectionRoleChanged.test.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  assert,
+  describe,
+  test,
+  clearStore,
+  beforeAll,
+  afterAll,
+} from 'matchstick-as/assembly/index';
+import { BigInt, Bytes } from '@graphprotocol/graph-ts';
+import { createNewCollectionRoleChanged, handleCollectionRoleChangedList, makeEventId, TOKEN_OWNER_ONE, TOKEN_OWNER_TWO } from '../helpers/utils';
+import { CollectionRoleChanged } from '../../../generated/FleekNFA/FleekNFA';
+
+describe('Collection Role Changed tests', () => {
+  beforeAll(() => {
+    // Collection Role Changed
+    let collectionRoleChangedList: CollectionRoleChanged[] = [];
+
+    collectionRoleChangedList.push(
+      createNewCollectionRoleChanged(0, 0, TOKEN_OWNER_ONE, true, TOKEN_OWNER_TWO) // Token Owner Two grants collection owner access to Token Owner One 
```

Comment:
The comment here is confusing.  How can a token owner grant collection owner perms to anyone?

Reply:
The name of the address const is token owner one/two. I see why that could be confusing. I will change them to user_one/two